tests/PropertySet_2.py

```diff
@@ -357,13 +357,11 @@ def testToDict(self):
 
         self.assertIsInstance(d2["bool"], bool)
         self.assertEqual(d2["bool"], True)
-        # Newer versions of SWIG can return int here
-        self.assertIsInstance(d2["short"], long)
+        self.assertIsInstance(d2["short"], int)
```

Comment:
Yes, driven by the SWIG update (when the code was first written we did not realize that we had to update SWIG).